### PR TITLE
LPD-44819 DDMFormValuesExportImportContentProcessor on importation of non ordered layouts giving NoSuchLayoutException when importing full site lar

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
LPD-44819 NoSuchLayoutException when importing full site .lar

# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-44819

On some corner cases, as the lar attached on the LPP, the layout parent are not on database so geting the breadcrumb to put on the ddm name giver a NoSuchLayoutException

# How am I fixing it?

On the case that the method getBreadcrumb throws an exception return the fallback value.

# How can you verify that it works?

Run the included automated tests.

# Commits

- **LPD-44819 on the case that the method getBreadcrumb throws an exception return the fallback value**
- **LPD-44819 change to package visibility to test purposes**
- **LPD-44819 check that if the getBreadcrumb method throws a NoSuchLayoutException the toJSON method returns the name of the json**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/exportimport/content/processor/LayoutImportDDMFormFieldValueTransformerTest.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+ */
+public class LayoutImportDDMFormFieldValueTransformerTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testToJSON() throws PortalException {
```
Comment (marcogalluzzi) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/exportimport/content/processor/LayoutImportDDMFormFieldValueTransformerTest.java:37]: This test name recalls a regular test with standard input values. I would give it a name to specify there is an exception involved.

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]
Everything looks good. Just a minor optional change, if you agree.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/exportimport/content/processor/LayoutImportDDMFormFieldValueTransformerTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/exportimport/content/processor/LayoutImportDDMFormFieldValueTransformerTest.java:37]: I will change the name of the test to :
`testToJSONWithNoSuchLayoutExceptionThrown`

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**